terraform/live/_envcommon/volume_delete.hcl

```diff
@@ -0,0 +1,7 @@
+terraform {
+
+  after_hook "delete volume hook" {
+    commands = ["apply"]
+    execute  = ["${dirname(find_in_parent_folders())}/../../../volume-hook.sh"]
+  }
+}
```

Comment:
Why do we need a separate file for this hook? Can't we just put it in `storage.hcl`? I think it's a better approach since this hook is strictly related to storage class and we won't need all this new `includes` in the queue `.hcl `files

Reply:
Agree, it can be put into `storage.hcl`. I will change it.